templates/home/index.html

```diff
@@ -317,7 +317,8 @@ <h1 style="margin: 0 0 10px; color: black;" data-aos="fade-up">Jagrati Orientati
         <div class="container">
             <!-- Icon -->
             <h1 data-aos="fade-up">Donate Us</h1>
-            <p data-aos="zoom-in" data-aos-delay="100" data-aos-duration="700" data-aos-anchor="#donate-us h1">Jagrati, an initiative of IIITians, promotes education in villages. Along with that, we try to develop cultural values in these students so that they may serve the nation. At Jagrati, we help these little kids to find their passion and shape their lives.
+            <p data-aos="zoom-in" data-aos-delay="100" data-aos-duration="700" data-aos-anchor="#donate-us h1"><em>We make a living by what we get.<br> We make a life by what we give. <br>-Winston Churchill</em> <br><br>At Jagrati, we promote education in villages and help these little kids to find their passion and shape their lives.</p>
```

Comment:
Don't write Winston Churchill and Squash your commits into one, I've already explained the procedure.

Reply:
Okay sure, I'm trying to squash them.